cmd/crictl/container.go

```diff
@@ -221,6 +221,10 @@ var containerStatusCommand = cli.Command{
 			Name:  "output, o",
 			Usage: "Output format, One of: json|yaml|table",
```

Comment:
@feiskyer Do you still think we should have the `table` format?
I think for inspect, we may just need `json|yaml`. Just like `crictl info`. See an example [here](https://github.com/kubernetes-incubator/cri-containerd/pull/445#issuecomment-348025737)
The problem is that timestamp in CRI is unix timestamp, we may want to do a conversion in crictl.

Reply:
I'm not a big fan of table, let's drop it in favor of machine readable output

Reply:
I'm not a big fan of table formats either, but as @Random-Liu says the unix timestamp is not human readable output:

```
crictl --runtime-endpoint /var/run/cri-containerd.sock inspect -v -o json 44025055b48da66cb67bfc221f800fe50f1256d3e22f1cf714294344b29d582e 
{
  "status": {
    "id": "44025055b48da66cb67bfc221f800fe50f1256d3e22f1cf714294344b29d582e",
    "metadata": {
      "name": "busybox",
      "attempt": 0
    },
    "state": "CONTAINER_CREATED",
    "createdAt": "1511994785429097447",
    "startedAt": "0",
    "finishedAt": "0",
    "exitCode": 0,
    "image": {
      "image": "docker.io/library/busybox:latest"
    },
    "imageRef": "docker.io/library/busybox@sha256:bbc3a03235220b170ba48a157dd097dd1379299370e1ed99ce976df0355d24f0",
    "reason": "",
    "message": "",
    "logPath": ""
  },
```
vs:
```
crictl --runtime-endpoint /var/run/cri-containerd.sock inspect -v 44025055b48da66cb67bfc221f800fe50f1256d3e22f1cf714294344b29d582e 
ID: 44025055b48da66cb67bfc221f800fe50f1256d3e22f1cf714294344b29d582e
Name: busybox
State: CONTAINER_CREATED
Created: 28 minutes ago
```

Reply:
We can do some conversion there, not a big deal

Reply:
Yeah, we should do some conversion. :)

Reply:
Agreed, table is not useful for inspect.

Reply:
default changed to json..

Reply:
so.. @Random-Liu @runcom @feiskyer wrt. "some conversion"  
I suggest converting the timestamps to RFC3339Nano 
"2006-01-02T15:04:05.999999999Z07:00"
We can do it here but I don't like changing the output values provided from the runtime when representing them in json/yaml form. Suggest we specify in CRI that time values in status response info map should be in said RFC3339Nano format. Thoughts?